### ISSUE TITLE
intel fortran test error

### ISSUE THREAD

**Issue by loriab** [PCMSolver/pcmsolver#37]
FYI, I get this compiling master with Intel 16.0.3. Suppressing the test for now.

```
/path/to/pcmsolver2/tests/Fortran_host/Fortran_host.f90(115): error #8209: If type specification is omitted, each element in an array-constructor must have the same type and kind type parameters.   [NR_NUCLEI]
      mep = nuclear_mep(nr_nuclei, charges, reshape(coordinates, (/ 3, nr_nuclei /)), &
-----------------------------------------------------------------------^
compilation aborted for /path/to/pcmsolver2/tests/Fortran_host/Fortran_host.f90 (code 1)
```

**Comment by robertodr** [comment]
It should be fixed by using `(/ 3_c_int, nr_nuclei /)` as second argument in the `reshape` function. I am actually surprised that this hasn't been caught before...

**Comment by robertodr** [comment]
Closed by #38
